static/bidder-info/minutemedia.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "https://pbs.minutemedia-prebid.com/pbs-mm"
+maintainer:
+  email: hb@minutemedia.com
+gvlVendorID: 918
```

Comment:
GVL 918 is registered as "SPORTORITY UK LTD". Please confirm this is correct for the Minute Media adapter.

Reply:
Confirmed